### DISCUSSION TITLE
How do I forward content from a private group? How to get links to private groups?

### DISCUSSION THREAD

**Discussion by 5536086** [aahnik/tgcf#499] [Q&A]
How do I forward content from a private group? How to get links to private groups?

**Comment by MobCode100** [comment]
You can login as a user, that is a member of that private group, and forward as usual?

**Reply by 5536086** [reply]
Returns an error after opening the task

messages had to be ignored                         
                             consecutively                                      
                    WARNING  Server sent a very new message  mtprotostate.py:216
                             with ID 7277534466881295361,                       
                             ignoring                                           
                    WARNING  Security error while unpacking mtprotosender.py:523
                             a received message: Too many                       
                             messages had to be ignored                         
                             consecutively                                      
                    WARNING  Server sent a very new message  mtprotostate.py:216
                             with ID 7277534466881296385,                       
                             ignoring                                           
                    WARNING  Security error while unpacking mtprotosender.py:523
                             a received message: Too many                       
                             messages had to be ignored                         
                             consecutively                                      
                    WARNING  Server sent a very new message  mtprotostate.py:216
                             with ID 7277534466881306625,                       
                             ignoring                                           
                    WARNING  Security error while unpacking mtprotosender.py:523
                             a received message: Too many                       
                             messages had to be ignored                         
                             consecutively                                      
                    WARNING  Server sent a very new message  mtprotostate.py:216
                             with ID 7277534466881308673,                       
                             ignoring                                           
                    WARNING  Security error while unpacking mtprotosender.py:523
                             a received message: Too many                       
                             messages had to be ignored                         
                             consecutively
